packages/neuron-wallet/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged & yarn test"
```

Comment:
`&` or `&&`? But I think test can remove from `pre-commit`, CI will check it.

Reply:
`&&` is correct.

CI usually blocked